pkg/plugins/manager/testdata/plugin-with-dist/plugin/plugin.json

```diff
@@ -0,0 +1,16 @@
+{
+  "type": "datasource",
+  "name": "Test",
+  "id": "test-datasource",
+  "backend": true,
+  "executable": "test",
+  "state": "alpha",
+  "info": {
+    "version": "2.0.0",
```

Comment:
FTR, the test is to not load this version and use the v1.0.0 in the dist folder.